src/rules/no_eval.rs

```diff
@@ -1,10 +1,14 @@
 // Copyright 2020 the Deno authors. All rights reserved. MIT license.
 use super::Context;
 use super::LintRule;
+use crate::swc_util::StringRepr;
+use swc_common::Span;
 use swc_ecmascript::ast::CallExpr;
 use swc_ecmascript::ast::Expr;
 use swc_ecmascript::ast::ExprOrSuper;
+use swc_ecmascript::ast::VarDeclarator;
 use swc_ecmascript::visit::noop_visit_type;
+use swc_ecmascript::visit::swc_ecma_ast::ParenExpr;
```

Comment:
```suggestion
use swc_ecmascript::ast::ParenExpr;
```

I'm not sure why `ParenExpr` is also exported from `swc_ecmascript::visit::swc_ecma_ast`, but if you don't have any special reason, it's better to import like this.